Source/DafnyCore/Backends/SinglePassCodeGenerator/SinglePassCodeGenerator.cs

```diff
@@ -29,7 +29,7 @@ public abstract partial class SinglePassCodeGenerator {
     // we can prefix it with "_".
     // However, for backends such as Rust which need special internal fields, we want to clearly
     // disambiguate between compiler-generated names and user-defined names, hence this prefix.
```

Comment:
Not worth extra CI runs to fix, but sounds like this comment isn't accurate any more?

Reply:
Not completely accurate, but still indicates why this prefix isn't hard-coded at least.